Makefile

```diff
@@ -0,0 +1,12 @@
+all: test
+
+clean:
+	find . -name '*.pyc' -exec rm \{\} \;
+
+flake8:
```

Comment:
Done.
Nevertheless, @fpagnoux do we want the tracker to know about the `--country_package`?